vector/build.gradle

```diff
@@ -308,7 +308,7 @@ dependencies {
     // Fix issue with Jitsi. Inspired from https://github.com/android/android-test/issues/861#issuecomment-872067868
     // Error was lots of `Duplicate class org.checkerframework.common.reflection.qual.MethodVal found in modules jetified-checker-3.1 (org.checkerframework:checker:3.1.1) and jetified-checker-qual-3.12.0 (org.checkerframework:checker-qual:3.12.0)
     //noinspection GradleDependency Cannot use latest 3.15.0 since it required min API 26.
-    implementation "org.checkerframework:checker:3.11.0"
+    implementation "org.checkerframework:checker:3.27.0"
```

Comment:
seems that the comment `Cannot use latest 3.15.0 since it required min API 26` above is outdated.